yt/yt/ytlib/scheduler/config.h

```diff
@@ -2116,6 +2116,7 @@ class TOperationFairShareTreeRuntimeParameters
     std::optional<double> Weight;
     TPoolName Pool;
     TJobResourcesConfigPtr ResourceLimits;
+    std::optional<TBooleanFormula> SchedulingTagFilter;
```

Comment:
Here in per tree runtime parameters SchedulingTagFilter is still presented

Reply:
But you can pass scheduling tag filter separately to `TFairShareTree::UpdateOperationRuntimeParameters` and avoid storing it in `TOperationFairShareTreeRuntimeParameters`